src/bin/parser.js

```diff
@@ -263,7 +250,6 @@ export default class Harmonic {
             metadata.lang = lang;
             metadata.default_lang = config.i18n.default === lang ? false : true;
```

Comment:
@jaydson @viniciusdacal Is it just me or this looks odd?
I've looked all the way back to v0.3.0 and this line was still the same. Looks like this has been wrong for a long time.
I'll invert this logic.